tests/test_filter.py

```diff
@@ -67,7 +67,11 @@ def test_filter_spec(filter_args, expected_filter_spec):
     # TODO: check other properties of filter_spec?
 
 
-@pytest.fixture(scope="module", params=list(GridType))
+vector_grids = [gt for gt in GridType if gt.name.startswith("VECTOR")]
```

Comment:
Parsing the name string does not seem like a robust way to detect vector laplacians. Instead we should somehow refactor our base data structures to store this information. In the meantime, simply hard-coding the grid types here would not be a bad way to go. There are not so many.